aqua/pinentry-mac/Portfile

```diff
@@ -50,41 +39,45 @@ pre-fetch {
     }
 }
 
-platform macosx {
-    notes   "
-             If you previously didn't have pinentry or gpg-agent installed \
-             and ${name} has been installed as a dependency of gpg-agent, \
-             you don't need to do anything to use ${name}.
+use_autoreconf          yes
+autoreconf.args         -fiv
+configure.args          --disable-ncurses --enable-maintainer-mode
+use_xcode               yes
 
-             If you want to switch to the GTK2, ncurses or Qt4 based pinentry \
-             program, please install pinentry with the correct use flags and \
-             follow the instructions below substituting the ${name} program \
-             path with ${prefix}/bin/pinentry.
+destroot {
+    copy ${worksrcpath}/macosx/pinentry-mac.app ${destroot}${applications_dir}/${name}.app
+}
 
+notes "
+    If you previously didn't have pinentry or gpg-agent installed \
+    and ${name} has been installed as a dependency of gpg-agent, \
+    you don't need to do anything to use ${name}.
 
-             If you previously had pinentry and gpg-agent installed and would \
-             like to switch to ${name}, you will have to set \
-             it as your pinentry program in \$HOME/.gnupg/gpg-agent.conf.
+    If you want to switch to the GTK2, ncurses or Qt4 based pinentry \
+    program, please install pinentry with the correct use flags and \
+    follow the instructions below substituting the ${name} program \
+    path with ${prefix}/bin/pinentry.
```

Comment:
MacPorts supports such options, via variants. And indeed, we should add variants for the most common use-cases:
* gtk2
* ncurses
* qt5

Reply:
On a related note: To avoid opportunistic use of installed components - and potentially undefined behavior, depending on what a user has installed - we should explicitly disable all options [that we don't want enabled].

Ditto for enabling those items which we _definitely want_ enabled.

Reply:
> MacPorts supports such options, via variants. And indeed, we should add variants for the most common use-cases:
> * gtk2
> * ncurses
> * qt5
>
> On a related note: To avoid opportunistic use of installed components - and potentially undefined behavior, depending on what a user has installed - we should explicitly disable all options [that we don't want enabled].
> 
> Ditto for enabling those items which we _definitely want_ enabled.

If you want, I'll add most of this, and then we can review and refine. I'll try to get to that today, if possible.

Reply:
Sure, sounds great, thanks! I'll be traveling for the next few days, so I won't be able to spend much time on this right away.